selfdrive/car/volkswagen/values.py

```diff
@@ -844,7 +848,7 @@ class VWCarInfo(CarInfo):
       b'\xf1\x875Q0909144T \xf1\x891072\xf1\x82\00521G00807A1',
     ],
     (Ecu.fwdRadar, 0x757, None): [
-      b'\xf1\x875Q0907567N \xf1\x890400\xf1\x82\00101',
+      b'\xf1\x875Q0907567N \xf1\x890400\xf1\x82\x0101',
```

Comment:
```suggestion
      b'\xf1\x875Q0907567N \xf1\x890400\xf1\x82\00101',
```

Since this radar firmware was the same from your car (only difference being the octal vs hex representation) let's keep the original instead of changing it.